roslibrust/src/lib.rs

```diff
@@ -105,6 +105,37 @@ pub use rosbridge::*;
 pub mod rosapi;
 
 #[cfg(feature = "ros1")]
-mod ros1;
-#[cfg(feature = "ros1")]
-pub use ros1::*;
+pub mod ros1;
+
+/// For now starting with a central error type, may break this up more in future
+#[derive(thiserror::Error, Debug)]
+pub enum RosLibRustError {
```

Comment:
I moved these here out of `rosbridge` module as I start to use concrete domain-specific error types.